easybuild/easyconfigs/h/Hadoop/Hadoop-TeraSort-on-local-filesystem.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-examples/src/main/java/org/apache/hadoop/examples/terasort/TeraSort.java b/hadoop-mapreduce-project/hadoop-mapreduce-examples/src/main/java/org/apache/hadoop/examples/terasort/TeraSort.java
```

Comment:
add a comment where this comes from and what it does (anything above this line is comment, no 'commenting out' char needed)

are you the author?